devel/docker/compose-services.yml

```diff
@@ -0,0 +1,54 @@
+version: '3'
+services:
+  wdb:
+    image: postgres:latest
+    container_name: waiverdb_database
+    restart: always
+    expose:
+      - "5432"
+    volumes:
+      - ./settings/restore_waiverdb.sh:/docker-entrypoint-initdb.d/restore_db.sh
+      - /tmp/waiverdb.dump:/docker-entrypoint-initdb.d/wdb_pgdata
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready -h localhost -U waiverdb -d waiverdb"]
+      interval: 5s
+      timeout: 30s
+      retries: 3
+
+  waiverdb:
+    image: "quay.io/factory2/waiverdb:latest"
+    container_name: waiverdb_service
+    entrypoint: /bin/sh -i -c "waiverdb wait-for-db && waiverdb db upgrade && /usr/bin/gunicorn-3 --bind 0.0.0.0:6544 --access-logfile=- --enable-stdio-inheritance waiverdb.wsgi:app"
+    volumes:
+      - ./settings/waiverdb-settings.py:/etc/waiverdb/settings.py
+    ports:
+     - "6544:6544"
+    user: '0'
+    depends_on:
+      - wdb
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:6544/healthcheck"]
+      interval: 5s
+      timeout: 30s
+      retries: 3
+
+  greenwave:
+    image: "quay.io/factory2/greenwave:latest"
+    container_name: greenwave_service
+    entrypoint: ["/bin/sh", "-c", "gunicorn-3 --workers 8 --bind 0.0.0.0:6545 --access-logfile=- --error-logfile=- --enable-stdio-inheritance greenwave.wsgi:app"]
+    volumes:
+      - ./settings/greenwave-settings.py:/etc/greenwave/settings.py
+      - ./settings/policy.yaml:/etc/greenwave/policies/fedora.yaml
+    ports:
+      - "6545:6545"
+    depends_on:
+      - waiverdb
+    healthcheck:
+      test: ["CMD", "curl", "-f", "http://localhost:6545/api/v1.0/about"]
+      interval: 5s
+      timeout: 30s
+      retries: 3
+
+networks:
+  default:
+    driver: bridge
```

Comment:
I seem to recall this file being available for download somewhere - is it possible for us to avoid checking it in to our repo and just using the existing one? I can't remember where it was, but if you also don't know I can try to find out.

Reply:
https://pagure.io/greenwave/blob/master/f/docker/docker-compose.yml

Reply:
Do you think we can use that file instead of keeping a copy here?

Reply:
@sebwoj did you check if we can download that file ? Maybe that would be part of the vagrant provisioning ?

Reply:
@sebwoj can we do this?

Reply:
Now this file is in the different location:
https://pagure.io/greenwave/blob/master/f/docker-compose.yml
But with this config, we have to use resultsDB(not used by Bodhi).
As far as I remember, ResultsDB preparation(downloading resultsDB DB dump + extacting + loading to docker) takes 30< min + 10 GB extra.

Reply:
Do you think we should continue using this previous version then which doesn't exist in the Greenwave repo anymore?

Reply:
It exists, someone just moved it from one location to another:
FROM:
`https://pagure.io/greenwave/blob/master/f/docker/docker-compose.yml`
TO:
`https://pagure.io/greenwave/blob/master/f/docker-compose.yml`

But I think that we should use our, custom one(this from my PR), because of reasons I mentioned in my previous comment.
My docker-compose.yml config was created based on this from GW repo, but I customized it for Bodhi needs.

Reply:
Fair enough.